corehq/apps/hqcase/utils.py

```diff
@@ -88,21 +88,6 @@ def get_case_by_domain_hq_user_id(domain, user_id, case_type):
             return case
 
 
-def get_callcenter_case_mapping(domain, user_ids):
```

Comment:
Minor, but I'd vote to remove this code immediately in a separate PR, since it's less certain how long it'll be before we can schedule the reindex